staging/src/k8s.io/cloud-provider/controllers/nodelifecycle/node_lifecycle_controller.go

```diff
@@ -162,9 +161,8 @@ func (c *CloudNodeLifecycleController) MonitorNodes() {
 				Namespace: "",
 			}
 
-			c.recorder.Eventf(ref, v1.EventTypeNormal,
-				fmt.Sprintf("Deleting node %v because it does not exist in the cloud provider", node.Name),
```

Comment:
I doubt anyone is watching this event reason since it's so verbose and it includes the name of the node. Updating the reason to `DeletingNode` makes a lot of sense. FYI @cheftako